### DISCUSSION TITLE
How to change the API Endpoint

### DISCUSSION THREAD

**Discussion by leviethung2103** [open-webui/open-webui#3927] [General]
As far as I know, the open-webui can be configured to Ollama server and OpenAI.

Currently, I've developed an FastAPI application using the langchain and OpenAI. I could create the similar endpoints like Ollama.

Is there any solutions oepn-webui can work with customized APIs ?

Thank you

**Comment by justinh-rahb** [comment]
Create a [Function](https://openwebui.com/functions) or [Pipeline](https://github.com/open-webui/pipelines) adapter for it 💪

**Comment by karthikcs** [comment]
> As far as I know, the open-webui can be configured to Ollama server and OpenAI.
> 
> Currently, I've developed an FastAPI application using the langchain and OpenAI. I could create the similar endpoints like Ollama.
> 
> Is there any solutions oepn-webui can work with customized APIs ?
> 
> Thank you

Hi @leviethung2103 , we you able to achieve this? I have a similar requirement, to use my own application's end point as LLMs in Open WebUI
